### PR TITLE
Implemented to create s3 in root directory and used the s3 in test-…

### PR DESCRIPTION
**Pull request by chubbymilo**
- From the root directory to create S3 bucket and dynamodb  table
- Under test-region, copy the created S3 bucket name into the backend to use the created S3 bucket to store states.

### CODE REVIEW

**Comment by chubbymilo** [comment]
Will look into how I can avoid copying the S3 bucket name, from the tutorial https://blog.gruntwork.io/how-to-manage-terraform-state-28f5697e68fa , it suggested to use  -backend-config and a file that contains the name of S3 bucket and region.

**Comment by chubbymilo** [comment]
I think I will need to clean this up a bit but will look into it tomorrow!

**Comment by chubbymilo** [comment]
One S3 bucket to hold states of two lambda from two different regions.

The flow:
- Firstly we will need to create S3 bucket, from the root directory, an S3 is created via main.tf
- Uncomment the backend block from root maint.tf and copy the S3 bucket name into it
- Navigate to the environment folder, test-west-region1 and copy the S3 bucket name into the backend block, spin up the infrastructure as usual, and repeat the same for test-west-region2.

Now the S3 bucket stores all three state files of the S3 bucket creation and lambda creation of two different regions.

File: environments/test-west-region1/main.tf [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+module "python_lambda" {
+  source                  = "../../lambda"
+  aws_region              = var.aws_region
+  lambda_source_file_path = "../../"
+  lambda_file_name        = "app"
+}
+
+terraform {
+  backend "s3" {
```
Comment (ryan-caarter) [inline environments/test-west-region1/main.tf]: You can pass backend to the same single terraform block at the top 😎

File: environments/test-west-region2/main.tf [hunk 2]
```diff
@@ -0,0 +1,31 @@
+terraform {
+   required_version = ">= 1.2.0"
+  required_providers {
```
Comment (ryan-caarter) [inline environments/test-west-region2/main.tf:3]: Running 'terraform fmt -recursive -diff' from the root directory before committing is a good habit to keep indentation consistent

Comment (chubbymilo) [reply environments/test-west-region2/main.tf:3]: My bad! Should have done it, will fix it up!

File: environments/test-west-region1/main.tf (hunk 1 above)
Comment (chubbymilo) [reply environments/test-west-region1/main.tf]: right! was not sure if I should as not clear on best practices.I will change it.

**Pull request merged**